root/static/scripts/edit/URLCleanup.js

```diff
@@ -2508,6 +2508,28 @@ const CLEANUPS = {
     ],
     type: LINK_TYPES.otherdatabases,
   },
+  'overture': {
+    match: [new RegExp('^(https?://)?overture\\.doremus\\.org/', 'i')],
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?overture\.doremus\.org\/$/, 'https://overture.doremus.org/');
```

Comment:
In the `match` expression, case-insensitive match, but here and below (`regexp.exec()`) case-sensitive, imho everything should be case-insensitive (at least for domain part).

Reply:
We don't seem to do that elsewhere, but might be a good idea. @yvanzo ?

Reply:
:+1:

Reply:
Case-sensivity of the function `clean` function is not specific to the rule `doremus`, there are 138 `clean` functions. Since this is a much larger issue, it seems better to be addressed globally, the ticket MBS-11331 has been created to track it.